src/WebJobs.Script.Grpc/Channel/GrpcWorkerChannel.cs

```diff
@@ -1677,13 +1677,13 @@ private void AddSample<T>(List<T> samples, T sample)
 
         private void AddAdditionalTraceContext(MapField<string, string> attributes, ScriptInvocationContext context)
         {
-            bool isOtelEnabled = false;
+            bool isOtelEnabled = _scriptHostOptions?.Value.TelemetryMode == TelemetryMode.OpenTelemetry;
             if (_scriptHostOptions?.Value.TelemetryMode == TelemetryMode.OpenTelemetry)
```

Comment:
Ah sorry, the suggestion is also to remove the lines after this.

Reply:
yes, I have removed it in my local changes.